calico_kubernetes/calico_kubernetes.py

```diff
@@ -356,9 +369,16 @@ def _get_container_info(self, container_name):
     def _get_container_pid(self, container_name):
         return self._get_container_info(container_name)["State"]["Pid"]
 
+    def _get_ipset_from_endpoint(self, endpoint):
```

Comment:
I broke this out when I was writing the CALICO_NETWORKING changes. Since its only getting called once I will get rid of this